src/Database/Traits/Sluggable.php

```diff
@@ -53,9 +53,9 @@ public function slugAttributes()
      * @param int $maxLength Maximum length for the slug not including the counter.
      * @return string The generated value.
      */
-    public function setSluggedValue($slugAttribute, $sourceAttributes, $maxLength = 240)
+    public function setSluggedValue($slugAttribute, $sourceAttributes, $maxLength = 175)
```

Comment:
Why does the max length have to be 175 instead of 190? @Nai4rus

Reply:
@LukeTowers for counter -> `$maxLength Maximum length for the slug not including the counter.`
I think because of this, the old value was 240, not 255.

Reply:
Good point